docs/user-guide/deploying-your-docs.md

```diff
@@ -77,15 +75,16 @@ where `<projectname>` is the name you used to register your project with PyPI.
 
 There are a few prerequisites for the above to work:
 
-1. You must be using [Setuptools] in your `setup.py` script ([Distutils] does not
-offer an `upload_docs` command).
-2. Your project must already be registered with PyPI (use `python setup.py register`).
-3. Your `mkdocs.yml` config file and your "docs" directory (value assigned to
-the [docs_dir] configuration option) are presumed to be in the root directory of
-your project alongside your `setup.py` script.
-4. It is assumed that the default value (`"site"`) is assigned to the [site_dir]
-configuration option in your `mkdocs.yaml` config file. If you have set a
-different value, assign that value to the `--upload-dir` option.
+1. You must be using [Setuptools] in your `setup.py` script ([Distutils] does
+   not offer an `upload_docs` command).
+1. Your project must already be registered with PyPI (use `python setup.py
+   register`).
+1. Your `mkdocs.yml` config file and your "docs" directory (value assigned to
+   the [docs_dir] configuration option) are presumed to be in the root directory
+   of your project alongside your `setup.py` script.
+1. It is assumed that the default value (`"site"`) is assigned to the [site_dir]
+   configuration option in your `mkdocs.yaml` config file. If you have set a
+   different value, assign that value to the `--upload-dir` option.
```

Comment:
Yeah, likewise. If there is a way to easily disable it I'd be happy to see that happen.